imgboard.php

```diff
@@ -70,6 +70,12 @@ function error($mes, $dest, $fancy = 0) {
         $del = new SaguaroDelete;
         $del->userDel();
         break;
+    case 'banned':
+        require_once(CORE_DIR . "/admin/bans.php");
+        $ban  = new Banish;
```

Comment:
Questionable indenting. 

I'm not one for enforcing style as long as it's semi-consistent but right now I'm really not feeling this.